src/features/protectedData/ProtectedData.tsx

```diff
@@ -54,15 +55,18 @@ export default function ProtectedData() {
           </Box>
           <Box sx={{ mx: 4, paddingBottom: 20 }}>
             <Grid container spacing={2}>
-              {currentData?.map(({ address, name, schema }) => (
-                <Grid item key={address}>
-                  <ProtectedDataCard
-                    id={address}
-                    title={name || 'Undefined'}
-                    schema={schema}
-                  />
-                </Grid>
-              ))}
+              {currentData?.map(
+                ({ address, name, schema, creationTimestamp }) => (
+                  <Grid item key={address}>
+                    <ProtectedDataCard
```

Comment:
some dates don't fit into the chip, fix style of `ProtectedDataCard`
![image](https://github.com/iExecBlockchainComputing/web3mail-usecase-demo/assets/26487010/1c5f4e12-4f44-41fe-9830-f67c67ecbd3a)

Reply:
<img width="269" alt="Capture d’écran 2023-06-26 à 17 25 26" src="https://github.com/iExecBlockchainComputing/web3mail-usecase-demo/assets/72495599/f6b2c65a-5054-4251-97b6-9795592c868b">